### DISCUSSION TITLE
OpenLoco website link error

### DISCUSSION THREAD

**Discussion by FenderBender2** [OpenLoco/OpenLoco#2668] [General]
Each time there is a new release, the accompanying Changelog and Release Announcement are updated. These can be viewed from links on the Dowload page. The link error occurs when there is a sub-release following a major release (e.g. 24.09.1) - the Release Announcement link results in a 404 error.

The problem is that the link is generated incorectly. For the example above the link points to https://openloco.io/news/2024/09/1/openloco-v24.09.1.html.

The Release Announcement link should actually point to https://openloco.io/news/2024/09/openloco-v24.09.1.html.

**Comment by AaronVanGeffen** [comment]
It's been temporarily fixed by the release of v24.10, but yes, the current approach is too simplistic.
